tree/bst.go

```diff
@@ -160,3 +160,117 @@ func (n *node) getBSTNodeWithValue(val int) *node {
 
 	return n.right.getBSTNodeWithValue(val)
 }
+
+// changes By Prerna 08/01/2022
+
+// 1. Is validate BST
+func (t *BinarySearchTree) ValidBST() {
+	if t.isBST() {
+		fmt.Println("Valid BST")
+
+	} else {
```

Comment:
Then the else branch can be removed and at the bottom you can just return false.